### PR TITLE
Fix Compatbility with Multicompany

### PR DESCRIPTION
**Pull request by mapiolca**
Before : Show all projects of all entities
Now : Show only projects that are allowed by permissions

### CODE REVIEW

File: htdocs/core/boxes/box_project.php [hunk 1]
```diff
@@ -99,6 +100,7 @@ public function loadBox($max = 5)
             $sql = "SELECT p.rowid, p.ref, p.title, p.fk_statut, p.public";
             $sql.= " FROM ".MAIN_DB_PREFIX."projet as p";
             $sql.= " WHERE p.fk_statut = 1"; // Only open projects
+			$sql .= " AND entity IN (".getEntity('projet').")"; // Only current entity or severals if permission ok
```
Comment (hregis) [inline htdocs/core/boxes/box_project.php]: @mapiolca 

better is : 
use "project" instead "projet" and use entity filter before other filters

```
$sql .= " WHERE entity IN (".getEntity('project').")"; // Only current entity or severals if permission ok
$sql.= " AND p.fk_statut = 1"; // Only open projects
```

Comment (mapiolca) [reply htdocs/core/boxes/box_project.php]: Done

**Comment by hregis** [comment]
@mapiolca 

for better sql request performance use entity filter after the WHERE

```
$sql .= " WHERE p.entity IN (".getEntity('project').")";
$sql.= " AND p.fk_statut = 1"; // Only open projects
```

File: htdocs/core/boxes/box_project.php [hunk 2]
```diff
@@ -99,6 +100,7 @@ public function loadBox($max = 5)
             $sql = "SELECT p.rowid, p.ref, p.title, p.fk_statut, p.public";
             $sql.= " FROM ".MAIN_DB_PREFIX."projet as p";
             $sql.= " WHERE p.fk_statut = 1"; // Only open projects
+			$sql.= " WHERE entity IN (".getEntity('project').")"; // Only current entity or severals if permission ok
```
Comment (hregis) [inline htdocs/core/boxes/box_project.php]: @mapiolca  no ! look

$sql.= " WHERE entity IN (".getEntity('project').")"; // Only current entity or severals if permission ok
$sql.= " AND p.fk_statut = 1"; // Only open projects

**Pull request merged**